core/unitOps.chainmail

```diff
@@ -1,7 +1,11 @@
 /**
- * Unitss are wrappers on extents that have been validated by an UnitOps, and
- * can be verified as having been issued by the UnitOps. They contain their
- * extent and a Label. The label identifies a particular assay.
+ * Units are asset descriptions and take the form of labeled extents. 
+ * UnitOps execute the logic of how units are changed when digital 
+ * assets are merged, separated, or otherwise manipulated. For
+ * instance, a deposit of 2 bucks into a purse that already has 3 bucks
+ * gives a new balance of 5 bucks. An empty purse has 0 bucks. UnitOps 
+ * relies heavily on ExtentOps, which manipulates the non-labeled
+ * portion.
  */
 struct Units (Extent, Label) {
```

Comment:
I'd reverse the order of the type parameters: `struct Units (Label, Extent)`

Reply:
Ok, sounds good, that's in